docs/topics/validation-report-errors.adoc

```diff
@@ -0,0 +1,31 @@
+[[validation_report_errors]]
+= Reported Errors When Running the Validation Report
+
+The validation report will report any errors encountered while running the rules and tests. The following includes a list of these errors and how to resolve them.
+
+[cols="1,2,2", options="header"]
+|====
+|Error Message
+|Description
+|Resolution
+
+|No test file matching rule
+|This error occurs when a rule file exists without a corresponding test file.
+|Create a test file for the existing rule, as outlined in xref:test_xml_rule[Testing XML Rules].
+
+|Test rule Ids __RULE_NAME__ not found!
+|This error is thrown when a rule exists without a corresponding ruletest.
+|Create a test for the existing rule, as outlined in xref:test_xml_rule_structure[Test XML Rule Structure].
+
+|XML parse fail on file __FILE_NAME__
+|The syntax in the XML file is invalid, and unable to be parsed successfully by the rule validator.
+|Correct the invalid syntax.
+
+|Hint for __RULE_NAME__ was not found!
+|The hint defined in the test rule does not match the executed rule's value.
+|Ensure the expected hint matches the returned hint.
+
+|iterable-_ATTRIBUTE_ not _VALUE_
```

Comment:
I think this could be replaced by:

* Error Message: `The rule with id="__RULE_ID__" has not been executed`
* Description: (just as example, improved as you wish) `The rule with the provided id has not been executed during the validation.`
* Resolution: (just as example, improved as you wish) `Please check there's a test data file that matches the condition in the mentioned rule.`